pages/10.contacts/03.manage-segments/docs.en.md

```diff
@@ -143,6 +143,14 @@ manually added will remain part of the list regardless of filters.
 In addition to segments you can also manually add any contact to a list by
 clicking the Preferences button at the segments tab, use the dropdown to select a segment and add the contact to it or click on the x next to a segment in the input field to remove the contact.
 
+## Add categories to segment
+
+Apart from organizing various Mautic elements, categories help with organizing contacts. To add categories to a segment, create a new [category](../categories#creating-managing-categories) of type `segment`. When you add a new segment, you can now sort and select from the categories you previously created.
+
+![Screenshot showing segment categories](add-category-segment.jpg)
+
+To determine whether your segment is linked to the categories you created, first create a new segment without any category assigned. From the Segments screen, click in the `Segment Category Filter` in the toolbar. Choose the category linked to your segment. Only the segment associated with that category is displayed, the other segments are not listed.
+
 ## Delete all contacts in a segment
 
 Filter the contacts in the segment. The batch delete action in the contact table allows deletion of up to 100 contacts at one time. This is a performance precaution since deleting more contacts at one time could cause issues. This feature can be used for hundreds of contacts.
```

Comment:
```suggestion
Filter the Contacts in the Segment. The batch delete action in the Contact table allows deletion of up to 100 Contacts at one time. This is a performance precaution since deleting more Contacts at one time could cause issues. This feature can be used for hundreds of Contacts.
```